js/edit.js

```diff
@@ -58,10 +57,14 @@
 
         link.appendChild(label);
 
-        var value = document.createElement('span');
-        value.textContent = currentSection[key];
-        value.dataset.id = key;
-        link.appendChild(value);
+        var valueElt = document.createElement('div');
+        var value = currentSection[key];
+        valueElt.setAttribute('aria-label', value);
+        valueElt.dataset.id = key;
+        valueElt.className = 'palette-item';
+        valueElt.style.backgroundColor = value;
```

Comment:
Now we show a palette item (a colored square) instead of the hexadecimal color value.